frontend/controller/actions/identity-kv.js

```diff
@@ -115,6 +115,37 @@ export default (sbp('sbp/selectors/register', {
       }
     })
   },
+  'gi.actions/identity/kv/markAsUnread': ({ contractID, messageHash, createdHeight, unreadMessages }: {
+    contractID: string,
+    messageHash: string,
+    createdHeight: number,
+    unreadMessages: Array<{ messageHash: string, createdHeight: number }>
+  }) => {
+    return sbp('okTurtles.eventQueue/queueEvent', KV_QUEUE, async () => {
+      const getUpdatedUnreadMessages = async () => {
+        const currentData = await sbp('gi.actions/identity/kv/fetchChatRoomUnreadMessages')
```

Comment:
This is OK, but will need to be updated after #2742 is merged (the signature will be updated and the fetch will no longer be needed)

Reply:
@corrideat Thanks for a heads up. It seems that PR has not been merged yet. Will make these updates if that goes earlier than this PR.